demo/px-datetime-range-field-demo.html

```diff
@@ -216,6 +217,12 @@
         inputType: 'toggle'
       },
 
+      fullWidth: {
+        type: Boolean,
+        defaultValue: false,
```

Comment:
just 'value' not 'defaultValue' (unless this is on purpose?).

Reply:
Just a typo

Reply:
Oh yeah... :slowclap: